src/components/CoverageFieldArray/CoverageFieldArray.js

```diff
@@ -20,10 +20,15 @@ class CoverageFieldArray extends React.Component {
     name: PropTypes.string.isRequired,
     onAddField: PropTypes.func.isRequired,
     onDeleteField: PropTypes.func.isRequired,
+    isEmptyCoverageMessage: PropTypes.string,
```

Comment:
Can you change the proptype to `isEmptyCoverageId` instead? as its an id but not the actual message